File: DayTwo/Java/LevelThree/TaskFour.java
import java.util.Scanner;

public class TaskFour{
    public static void main(String... args){
    
    Scanner input = new Scanner(System.in);
    
    System.out.print("Enter a number: ");
    int number = input.nextInt();
    
    int result = 0;
        
        for(int count = 1; count <= 12; count++){
        result = count * number;
            System.out.println(number +  " x " + count + " = " + result);
        }
    }
}

File: DayTwo/JavaScript/LevelThree/TaskFour.js
const prompt = require("prompt-sync")();

let number = Number(prompt("Enter a number: "))

let result = 0;

    for(let count = 1; count <= 12; count++){
        result = count * number;
            console.log(number +  " x " + count + " = " + result);
            
     }
